SonarQube.Common/ProcessRunner.cs

```diff
@@ -25,6 +25,15 @@ public sealed class ProcessRunner
         public int ExitCode { get; private set; }
 
         /// <summary>
+        /// Runs the specified executable without timeout 
+        /// </summary>
+        /// <returns>True if the process exist code is 0</returns>
```

Comment:
Typos. "True if the process exited successfully, otherwise false" would be better.